src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+  },
+  'use ::ng-deep instead of /deep/': {
+    action:
+      'Thay thees `/deep/` bằng `::ng-deep` trong các file Styles của bạn, [đọc thêm về angular component styles và ::ng-deep](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep). `/deep/` và `::ng-deep` đều là những API bị khai tử nhưng việc dùng `::ng-deep` được khuyến khích hơn cho đến khi "shadow-piercing descendant" bị [xoá bỏ khỏi các trình duyện và công cụ liên quan](https://www.chromestatus.com/features/6750456638341120) hoàn toàn.'
+  },
+  'TypeScript 3.4': {
+    action:
+      'Angular giờ đã sử dụng TypeScript 3.4, [đọc thêm về các lỗi có thể xảy ra từ việc kiểm tra kiểu dữ liệu đã được cải thiện](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html).'
+  },
+  'node 10': {
+    action:
+      'Bảo đảm rằng bạn đang sử dụng <a href="http://www.hostingadvice.com/how-to/update-node-js-latest-version/" target="_blank">Node 10 hoặc mới hơn</a>'
+  },
+  'Differential Loading': {
+    action:
+      'Lệnh "build" của Angular CLI giờ sẽ tự động khởi tạo một bản ES2015 đời mới với các Polyfills tối thiểu và một bản ES5 đời cũ hơn để tương thích với các trình duyệt cũ, cùng với việc sẽ tải đúng loại build nào (ES2015 hoặc ES5) tuỳ thuộc vào trình duyện. Bạn có thể rút ra khỏi việc thay đổi này bằng cách cấu hình `target` sang `es5` trong `tsconfig.json` của bạn. Đọc thêm tại [angular.io](https://angular.io/guide/deployment#differential-loading).'
+  },
+  'CLI Telemetry': {
+    action:
+      'Khi sử dụng các phiên bản mới của Angular CLI, bạn sẽ được hỏi là nếu bạn có muốn tham gia vào việc chia sẻ dữ liệu sử dụng CLI cho Angular hay không. Bạn cũng có thể thêm vào tài khoản Google Analytics của bản thân. Việc này sẽ giúp cho chúng tôi có được những quyết định đúng đắn hơn về tính năng nào của Angular CLI cần được ưu tiên, và sẽ giúp chúng tôi đo đạc được sức ảnh hưởng của chúng tôi lên cộng đồng. Đọc thêm tại [angular.io](https://angular.io/analytics).'
+  },
+  'static query timing': {
+    action:
+      'Nếu bạn sử dụng `ViewChild` hoặc `ContentChild`, chúng tôi thay đổi cách chúng tôi những phép truy vấn này được xử lý để mang đến cho lập trình viên nhiều quyền kiểm soát hơn. Bạn phải tự xác định xem Change Detection nên thực thi trước khi kết quả được trả về. Ví dụ:  `@ContentChild(\'foo\', {static: false}) foo !: ElementRef;`. `ng update` sẽ tự động cập nhật các phép truy vấn này cho các bạn. Đọc thêm tại [angular.io](https://angular.io/guide/static-query-migration).'
+  },
+  'v8 material update': {
+    action: 'Nâng cấp Angular Material lên phiên bản 8 bằng việc thực thi lệnh `ng update @angular/material@8` trên terminal của bạn.',
+  },
+  'deep imports': {
+    action:
+      'Thay vì import từ `@angular/material`, bạn nên import cụ thể Component nào bạn muốn sử dụng. Ví dụ: `@angular/material/button`. `ng update` sẽ làm điều này cho bạn.'
```

Comment:
"chính xác" may be a better word than "cụ thể" here

Reply:
Good suggestion!